targets/playground/examples/thymio-default-behaviours.aesl

```diff
@@ -81,6 +81,9 @@ var dir
 var rc5_speed_l
 var rc5_speed_t
 
+var kernel1[5] = [1,2,3,2,1]
+var kernel2[5] = [-4, -3, 1, 3, 4]
```

Comment:
Be consistent with spacing between `kernel1` and `kernel2`. Also, maybe there are more relevant names, such as `avoidanceKernelX`, or even by replacing `1` and `2` with their semantic meaning (@mbonani might be able to give some explanation about the semantics).